### PR TITLE
Add Caseys Translation and Messaging tests to github

### PR DESCRIPTION
**Pull request by btswann11**

### CODE REVIEW

File: test-plans/message-send-and-receive.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+### Test: Message Sent Shows Status Indicators
+
+This test covers requirements: FR6, FR7, FR8
```
Comment (btswann11) [inline test-plans/message-send-and-receive.md:3]: can we write down what these requirements are

File: test-plans/message-send-and-receive.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+2. Type "Hello from User A" into the message input box  
+3. Click the **Send** button  
+4. Confirm that the message is initially labeled **In Progress**  
+5. Wait for backend confirmation  
+6. Confirm that the message status changes to **Sent**
+
+### Expected Results:
+- The message enters the system and displays the **In Progress** label after pressing Send  
+- Once acknowledged by the backend, the label changes to **Sent**  
+- The user sees status updates reflected in the chat interface
```
Comment (btswann11) [inline test-plans/message-send-and-receive.md:16]: we can integrate expected results into the steps (ex, 1. press send. 2. verify that the system displays in progress)

File: test-plans/message-send-and-receive.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+### Acceptance Criteria:
+- 90% or more of messages must progress from **In Progress** to **Sent** within 5 seconds  
+- No messages should skip status labels or go out of order  
+- Less than 10% failure rate is acceptable due to backend instability or edge cases
+
+### Notes:
+This test assumes message input and backend status handling are already implemented. This test does not cover message reception or translation.
```
Comment (btswann11) [inline test-plans/message-send-and-receive.md:24]: we have a blanket acceptance criteria so that part can be removed. We dont need to assume anything because the tests themselves assume that the modules are already working.
```suggestion
```

File: test-plans/translation.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
+### Test: Verify Message Translation Between Languages
```
Comment (btswann11) [inline test-plans/translation.md:1]: same comments on this file as the other one

File: test-plans/translation.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+This test covers requirement: FR11
```
Comment (matt-p-dorrington) [inline test-plans/translation.md:3]: FR11 - Show Translated Message - The users local screen shall show original message (source language) and the translated version (destination language)

File: test-plans/message-send-and-receive.md (hunk 1 above)
Comment (Blueboybass) [reply test-plans/message-send-and-receive.md:3]: Updated

File: test-plans/message-send-and-receive.md (hunk 2 above)
Comment (Blueboybass) [reply test-plans/message-send-and-receive.md:16]: Updated

File: test-plans/message-send-and-receive.md (hunk 3 above)
Comment (Blueboybass) [reply test-plans/message-send-and-receive.md:24]: Updated

File: test-plans/translation.md (hunk 4 above)
Comment (Blueboybass) [reply test-plans/translation.md:1]: Updated

File: test-plans/translation.md (hunk 5 above)
Comment (Blueboybass) [reply test-plans/translation.md:3]: Added

File: test-plans/translation.md [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+1. Log in as **User A** with preferred language set to **English**  
+2. Log in as **User B** with preferred language set to **Spanish**  
+3. User A sends the message: `"Hello, how are you?"`  
+4. Verify that **User B** receives the translated message in **Spanish**  
+5. User B sends a message in Spanish: `"Estoy bien, gracias."`  
+6. Verify that **User A** receives the translated message in **English**  
+7. Log in as **User C** with preferred language set to **French**  
+8. User B sends a message to **User C**  
+9. Verify that **User C** receives the translated message in **French** (translated from Spanish)  
+10. Verify that all translations are accurate, meaningful, and completed within 5 seconds
```
Comment (btswann11) [inline test-plans/translation.md:23]: one last comment: since the `5 seconds` thing is another requirement, we can either add that to the list of applicable tests, or we can remove it from this line. Accurate and meaningful is also arbitrary but im lazy so we can leave it as is. So the TLDR is just- remove the 5 seconds from here and then we can test that somewhere else (because testing both of them is more like integration or acceptance testing)
```suggestion
10. Verify that all translations are accurate, meaningful, and completed
```
